great_expectations/core/expectation_configuration.py

```diff
@@ -784,6 +784,42 @@ class ExpectationConfiguration(SerializableDictDot):
                 "catch_exceptions": False,
             },
         },
+        ####
+        "expect_file_line_regex_match_count_to_be_between": {
```

Comment:
What is "file_line_regex"?..

Reply:
There was an entire class of file-based expectations that was missing from the kwargs lookup dict in `expectation_configuration.py`.

Reply:
@roblim I was basically wondering -- what is the meaning of "file_line" or is it "line_regex", or is it "file_line_regex"?..  what is the meaning of this term?  Is it explained in a documentatioon?

Reply:
Docstring for `great_expectations.data_asset.file_data_asset.FileDataAsset.expect_file_line_regex_match_count_to_be_between`:
```
"""
        Expect the number of times a regular expression appears on each line of
        a file to be between a maximum and minimum value.

        Args:
            regex: \
                A string that can be compiled as valid regular expression to match

            expected_min_count (None or nonnegative integer): \
                Specifies the minimum number of times regex is expected to appear
                on each line of the file

            expected_max_count (None or nonnegative integer): \
               Specifies the maximum number of times regex is expected to appear
               on each line of the file
        Keyword Args:
            skip (None or nonnegative integer): \
                Integer specifying the first lines in the file the method should
                skip before assessing expectations

            mostly (None or number between 0 and 1): \
                Specifies an acceptable error for expectations. If the percentage
                of unexpected lines is less than mostly, the method still returns
                true even if all lines don't match the expectation criteria.

            null_lines_regex (valid regular expression or None): \
                If not none, a regex to skip lines as null. Defaults to empty or whitespace-only lines.
"""
```

Reply:
All good!  Thanks, @roblim !